julian_dateparser.py

```diff
@@ -173,8 +173,12 @@
                     | Combine(Literal("3")  + Word("012345",nums,exact=2))
                     | Combine(Literal("36") + Word("0123456",exact=1))
                   ) + WordEnd(nums)
+# This appears to have stopped working in Python 3.9 and 3.10
```

Comment:
Do you want to leave this comment here in case it's fixed in the future? If not you can remove it.